src/popover/Popover.js

```diff
@@ -76,6 +76,9 @@ export default class Popover extends React.Component {
   componentWillUnmount() {
     this.removePopover();
     this.removeListeners();
+    this.onMouseEventDebouncedWhenOpen.cancel();
```

Comment:
@sawsanias looking at the code I noticed that `this.onMouseEventDebouncedWhenOpen/Closed` are dynamically defined [here](https://github.com/buildo/react-components/blob/a7acbe0256791373d2e1ca270424e3997922523d/src/popover/Popover.js#L300).

what happens if they're `undefined`? don't these functions end up in a new uncaught `TypeError`?

**EDIT:**
I tried and it doesn't throw any error, this means that we're instantiating `this.onMouseEventDebouncedWhenOpen/Closed` even when we don't need them.

@sawsanias maybe we could fix it directly in this PR?

I'm talking about [this](https://github.com/buildo/react-components/blob/a7acbe0256791373d2e1ca270424e3997922523d/src/popover/Popover.js#L300): if `delayWhenOpen/Closed` is missing `this.onMouseEventDebouncedWhenOpen/Closed` shouldn't be defined.

After this fix I would also call `this.onMouseEventDebouncedWhenOpen/Closed.cancel()` only if they're not `undefined`